pkgs/by-name/mo/monaspace/package.nix

```
@@ -58,4 +33,35 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     maintainers = [ ];
     platforms = lib.platforms.all;
   };
+  makeFont =
+    font:
```

I think that this makes it clearer what the function depends on directly instead of using implicit variables from `font`.
```suggestion
    font:
    {
       pname,
       fontFamily,
       variant,
       version,
       baseUrl,
       hash,
       destination,
       meta,
    }:
```